docs/kubernetes/kind.mdx

```diff
@@ -0,0 +1,185 @@
+```
+
+Validate the installation with:
+
+```shell
+# make sure pods are ready
+kubectl rollout status deploy -l app=wasmcloud-operator -n wasmcloud-operator
+# apiservice should be available
+kubectl get apiservices.apiregistration.k8s.io v1beta1.core.oam.dev
+```
+
+### Create wasmCloud host
+
+Apply the `wasmcloud-host` manifest:
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/wasmcloud-host.yaml
+```
+
+Check wasmCloud host status:
+
+```shell
+kubectl describe wasmcloudhostconfig wasmcloud-host
+```
+
+### Run a WebAssembly component on Kubernetes
+
+When you `kubectl apply` a [wasmCloud application manifest](../../docs/ecosystem/wadm/model), the cluster automatically provisions the component workload with wasmCloud. 
+
+This example uses the [`hello-world-application.yaml` manifest](https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/hello-world-application.yaml) included in the operator's quickstart. (The [source code for the application](https://github.com/wasmCloud/wasmCloud/tree/main/examples/rust/components/http-hello-world) is available in the wasmCloud repository.)
+
+Below is an excerpt of the manifest:
+
+```yaml {7-7}
+...
+spec:
+  components:
+    - name: http-component
+      type: component
+      properties:
+        image: ghcr.io/wasmcloud/components/http-hello-world-rust:0.1.0
+...
+```
+The component is packaged as an OCI artifact and specified in the `image` field. This isn't a container, but a component conforming to OCI standards, meaning that it can be used with existing registries for container images. 
+
+Run `kubectl apply`: 
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/wasmCloud/wasmcloud-operator/main/examples/quickstart/hello-world-application.yaml
+```
+
+View the deployment status:
+
+```shell
+kubectl get application
+```
+```shell
+APPLICATION   DEPLOYED VERSION   LATEST VERSION   STATUS
+hello-world   v0.0.1             v0.0.1           Deployed
+```
+
+### Manage applications with wash
+
+To connect `wash` to the cluster, port-forward into the NATS service running in your Kubernetes cluster. (Note: `4222` is the port for the NATS service, `4223` is the port for NATS websockets.)
+
+```shell
+kubectl port-forward svc/nats 4222:4222 4223:4223
+```
+Now you can connect to wasmCloud on Kubernetes with your local wash toolchain:
+
+```shell
+wash app list
+```
+
+View the hello world component running on the Kubernetes cluster, manageable via our local `wash` CLI:
+
+```shell
+Name          Latest Version   Deployed Version   Deploy Status  Description                                                                                                  
+hello-world   v0.0.1           v0.0.1                  Deployed  HTTP hello world demo in Rust, using the WebAssembly Component Model and WebAssembly Interfaces Types (WIT)
+```
+
```

Comment:
We should add testing and using the app sections. Then also a debug section

Now to call and test the application:
```bash
kubectl port-forward svc/wasmcloud-host 8080
curl localhost:8080
```

## Debugging

Get host logs:
```bash
k logs -l app.kubernetes.io/instance=wasmcloud-host -c wasmcloud-host
```

Common mistakes:
- Deploying with a local file reference instead of OCI